src/release-pr.ts

```diff
@@ -111,11 +98,10 @@ export class ReleasePR {
     this.repoUrl = options.repoUrl;
     this.token = options.token;
     this.path = options.path;
-    this.packageName = options.packageName;
+    this.packageName = options.packageName || '';
```

Comment:
I'm ok with an empty string if you don't think the following is appropriate (moved further down where `this.{gh,releaseType}` are available):

```suggestion
    this.packageName = options.packageName || await factory
      .releasePRClass(this.releaseType)
      .lookupPackageName(this.gh, this.path);
```